js/apps/admin-ui/src/clients/authorization/policy/ClientScope.tsx

```diff
@@ -162,9 +148,6 @@ export const ClientScope = () => {
           </Tbody>
         </Table>
       )}
-      {errors.clientScopes && (
-        <FormErrorText message={t("requiredClientScope")} />
```

Comment:
Looks like this is the only place the `requiredClientScope` label was used, so it can be removed from the properties files as well.